cfgov/unprocessed/apps/paying-for-college/js/util/other-utils.js

```diff
@@ -56,6 +56,19 @@ function getStateByCode( code ) {
   return statesByCode[code];
 }
 
+function _getMatchesMethod( el ) {
+  return el.matches ||
+         el.webkitMatchesSelector ||
+         el.mozMatchesSelector ||
+         el.msMatchesSelector;
+}
+
+function selectorMatches( el, selector ) {
```

Comment:
Run `gulp lint --fix` and add jsdocs to these two methods to fix warnings.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5869/commits/f336de3ba8d47610a664c77abfdfcf759a02118b